code/addons/docs/src/types.ts

```diff
@@ -0,0 +1,223 @@
+import type { ModuleExport, ModuleExports } from '@storybook/types';
+
+type StoryBlockParameters = {
```

Comment:
Can you apply doc blocks at this level?

```suggestion
/**
 * @see https://storybook.js.org/docs/api/doc-blocks/doc-block-story
 */
type StoryBlockParameters = {
```

Reply:
Unfortunately not, I tried